src/v/kafka/server/protocol.cc

```diff
@@ -100,6 +100,13 @@ ss::future<> protocol::apply(rpc::server::resources rs) {
     return ss::do_until(
              [ctx] { return ctx->is_finished_parsing(); },
              [ctx] { return ctx->process_one_request(); })
+      .handle_exception([ctx](std::exception_ptr propogate) {
+          vlog(
+            klog.info,
+            "Auth failed on ip: {}",
+            ctx->client_host().as_ipv4_address());
```

Comment:
what if ipv6? why not `client_host()` only?

Reply:
I did not know client_host() (aka `inet_address`) could be printed. I will try that out.